Content.Shared/Preferences/HumanoidCharacterProfile.cs

```diff
@@ -43,13 +50,12 @@ public sealed partial class HumanoidCharacterProfile : ICharacterProfile
             Sex sex,
             Gender gender,
             HumanoidCharacterAppearance appearance,
-            ClothingPreference clothing,
-            BackpackPreference backpack,
             SpawnPriorityPreference spawnPriority,
             Dictionary<string, JobPriority> jobPriorities,
             PreferenceUnavailableMode preferenceUnavailable,
             List<string> antagPreferences,
             List<string> traitPreferences,
+            Dictionary<string, RoleLoadout> loadouts)
```

Comment:
Expects a ; at the end of the line here, might be what's causing issues?

Reply:
No, the ) at the end of this line should be a ,